ci-operator/config/openshift/ceph-csi/openshift-ceph-csi-master.yaml

```diff
@@ -0,0 +1,23 @@
+binary_build_commands: '"make containerized-build"'
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.15
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- artifact_dir: /tmp/artifacts
+  as: containerized-test
+  commands: '"CONTAINER_CMD=docker make containerized-test"'
```

Comment:
No need to quote this, I think it will even lead to it being treated as a command to execute including the spaces

Reply:
this has been addressed